skule_vote/backend/views.py

```diff
@@ -9,13 +9,20 @@
 from django.urls import reverse_lazy
 from django.views import View
 from rest_framework import exceptions, generics
+from rest_framework.generics import CreateAPIView
```

Comment:
We have already imported `generics` in the line above. Remove this line and then subclass from `generics.CreateAPIView` like we do for the other classes